frontend/src/v5/ui/controls/inputs/datePicker/baseCalendarPicker/baseCalendarPicker.component.tsx

```diff
@@ -46,7 +46,7 @@ export const BaseCalendarPicker = ({
 	return (
 		<PickerComponent
 			{...props}
-			value={value}
+			value={value || (defaultValue ? dayjs(defaultValue) : null)}
```

Comment:
I dont think is right to assume that everything with a default value is a date. Isnt there another place we can take this decision where we safely know is a date?